src/core/dbus-unit.c

```diff
@@ -1442,6 +1442,34 @@ static int property_get_io_counter(
         return sd_bus_message_append(reply, "t", value);
 }
 
+static int property_get_effective_limit(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        static const char *const table[_CGROUP_LIMIT_TYPE_MAX] = {
+                [CGROUP_LIMIT_MEMORY_MAX]  = "EffectiveMemoryMax",
+                [CGROUP_LIMIT_MEMORY_HIGH] = "EffectiveMemoryHigh",
+                [CGROUP_LIMIT_TASKS_MAX]   = "EffectiveTasksMax",
+        };
+
+        uint64_t value = CGROUP_LIMIT_MAX;
+        Unit *u = ASSERT_PTR(userdata);
+        ssize_t type;
+
+        assert(bus);
+        assert(reply);
+        assert(property);
+
+        assert_se((type = string_table_lookup(table, ELEMENTSOF(table), property)) >= 0);
```

Comment:
Please define the table and lookup function in `core/cgroup.[ch]`, as with all existing `Metrics`